pkgs/development/python-modules/gpib-ctypes/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, pytestCheckHook
+, pytest-runner
+, linux-gpib
+}:
+
+buildPythonPackage rec {
+  pname = "gpib_ctypes";
+  version = "0.3.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-c9l6TNmM4PtbvopnnFi5R1dQ9o3MI39BHHHPSGqfjCY=";
+  };
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-runner
```

Comment:
```suggestion
```
`pytest-runner` is not needed, in fact.
We can remove this from setup.py in postPatch.

Reply:
How can I remove that from postPatch?

Reply:
with
```
substituteInPlace setup.py \
  --replace "'pytest-runner'," ""
```

Reply:
Updated the PR. Thanks